src/test/resources/features/foi/FOIDispatch.feature

```diff
@@ -10,6 +10,7 @@ Feature: FOI Dispatch
     And I select "Email" as the response
     And I select "Information released in full" as outcome of the case
     And I click the "Continue" button
+    And I click the "Confirm" button
```

Comment:
We have a goal of making the scenarios more Declarative and less Imperative, so that they can better function as living documentation for the wider team. 
Instead of adding another 'I click the x button' statement, could we look at making the language of these scenarios less about DECS interactions and more about users intentions?
For example, maybe include the button click in the previous step, or maybe create a step that indicates what the user is actually confirming? 
I found creating a unique step useful in places as it meant I could add a wait for a new page title where we had multiple clicks of the same button in succession, which was causing stale element exceptions because of race conditions.